src/i18n/es.js

```diff
@@ -34,12 +34,20 @@ const urlMasterSubscriber = `${urlDopplerLegacy}/Lists/MasterSubscriber/`;
 
 export default {
   change_plan: {
+    all_of_plan: `Todo lo de {plan} más:`,
     calculate_price: 'Calcular Precio',
+    cancel_campaign: 'Cancelar Campaña',
+    compare_features: 'Comparar Funcionalidades',
     description: '¿Realizas envíos periodicamente o un newsletter al mes?',
+    email_parameter: 'Parametros de email',
+    features: 'Funcionalidades:',
     per_month: 'por mes',
     recommended: 'Recomendado',
     selected_type: 'Planes por: ',
+    shipping_limit: 'Límite de envíos',
     since: 'Desde',
+    site_tracking: 'Comportamiento en Sitio',
+    smart_campaigns: 'Campañas inteligentes',
```

Comment:
```suggestion
    smart_campaigns: 'Envíos Inteligentes',
```
![image](https://user-images.githubusercontent.com/2439363/91482943-425a6d80-e87d-11ea-9bd7-c5a22d4706cc.png)